src/renderer/components/+workloads-deployments/deployments.store.ts

```diff
@@ -67,6 +68,10 @@ export class DeploymentStore extends KubeObjectStore<Deployment> {
       .getByLabel(deployment.getTemplateLabels())
       .filter(pod => pod.getNs() === deployment.getNs());
   }
+
+  getRelatedReplicas(deployment: Deployment) {
+    return replicaSetStore.getReplicaSetsByOwner(deployment);
+  }
```

Comment:
Would you mind doing this in a different PR? I don't see why it is necessary for rollout support.